src/core/icons/fileIcons.ts

```diff
@@ -395,22 +395,53 @@ export const fileIcons: FileIcons = {
       name: 'zip',
       fileExtensions: [
         'zip',
+        'z',
         'tar',
         'gz',
         'xz',
+        'lz',
+        'liz',
         'lzma',
+        'lzma2',
         'lz4',
+        'lz5',
+        'lzh',
+        'lha',
         'br',
         'bz2',
         'bzip2',
         'gzip',
         'brotli',
         '7z',
+        '001',
         'rar',
+        'far',
         'tz',
+        'taz',
+        'tlz',
         'txz',
         'tgz',
+        'tpz',
+        'tbz',
+        'tbz2',
         'zst',
+        'zstd',
+        'tzst',
+        'tzstd',
+        'cab',
+        'cpio',
+        'rpm',
+        'deb',
```

Comment:
Makes sense, but I'm thinking of creating a separate icon for deb packages

```suggestion
```

Reply:
but...

Reply:
deb is also archive

Reply:
Plenty of these extensions could potentially separate icon; we do have `verilog` and `disc` after all. This was more about getting the extensions added outright

Reply:
https://github.com/user-attachments/assets/48478018-3d3f-4c5f-8ed0-dd691419f180